content/blog/2021-02-18-dvc-2-0-pre-release.md

```diff
@@ -0,0 +1,282 @@
+---
+title: DVC 2.0 Pre-Release
+date: 2021-02-17
+description: |
+  Today, we're announcing DVC 2.0 pre-release. We'll share learnings from our
+  journey and how these will be reflected in the coming release.
+
+descriptionLong: |
+  The new release is a result of our learnings from our users. There are four
+  major features are coming:
+
+  🔗 ML pipelines templating and iterative foreach stages
+
+  🧪 Lightweight ML experiments
+
+  📍 ML model checkpoints
+
+  📈 Dvc-live - metrics logging
+
+picture: 2021-02-18/dvc-2-0-pre-release.png
+pictureComment: DVC 2.0 Pre-Release
+author: dmitry_petrov
+commentsUrl: https://discuss.dvc.org/t/dvc-3-years-anniversary-and-1-0-pre-release/374
+tags:
+  - Release
+  - MLOps
+  - DataOps
+---
+
+## Install
+
+First things first. You can install the 1.0 pre-release from the master branch
+in our repo (instruction [here](https://dvc.org/doc/install/pre-release)) or
+through pip:
+
+```bash
+pip install --upgrade --pre dvc
+```
+
+## ML pipelines parametrization and foreach stages
```

Comment:
foreach -> `foreach`

Reply:
I don't mind either way but here's the easy commit suggestion:
```suggestion
## ML pipelines parametrization and `foreach` stages
```